### PR TITLE
Update gpcheckcat summary report

### PR DESCRIPTION
**Pull request by stephendotcarter**
Currently to check previous runtimes it takes multiple grep commands to summarize the log file. This change adds all the information to a single line making it far easier to check historical runtimes. The historical runtimes can be used to estimate the required maintenance window for future runs.

### CODE REVIEW

**Comment by ivannovick** [comment]
This looks good to me @tangp3 can you please help review and push this as you have updated this file recently before?

**Comment by ivannovick** [comment]
@tangp3 ping can you review this and merge it?

**Comment by danielgustafsson** [comment]
Looks good to me, any script expecting and grepping for the previous output will have to altered though. Would this be common enough to warrant an note for upgrades?

**Comment by ivannovick** [comment]
i think for a major upgrade its fine

On Fri, Dec 18, 2015 at 5:03 AM, Daniel Gustafsson <notifications@github.com

File: gpMgmt/bin/lib/gpcheckcat [hunk 1]
```diff
@@ -4695,8 +4695,10 @@ def checkcatReport():
     myprint('')
     myprint('SUMMARY REPORT')
     myprint('===================================================================')
-    elapsed = str(datetime.timedelta(seconds=GV.elapsedTime))[:-4]
-    myprint('Total runtime for %d test(s): %s' % (GV.totalTestRun, elapsed))
+    elapsed = str(datetime.timedelta(seconds=GV.elapsedTime)).split('.')[0]
+    endTime = str(datetime.datetime.now().strftime('%Y-%m-%d %H:%I:%S'))
```
Comment (asimrp) [inline gpMgmt/bin/lib/gpcheckcat]: I think the `str()` is redundant here.  `strftime()` returns a string.

Comment (asimrp) [inline gpMgmt/bin/lib/gpcheckcat]: Can we replaces lines 4698 and 4699 with the following.  It is error prone to play with string representation of objects.
```python
elapsed = datetime.timedelta(seconds=int(GV.elapsedTime))
endtime = datetime.datetime.now().strftime(...)
```

Comment (stephendotcarter) [inline gpMgmt/bin/lib/gpcheckcat]: I've made the recommended changes.
I do agree it's more error prone but I think truncating the elapsed time to remove the microseconds makes it easier for the user to read the output. Nobody cares about or needs microsecond precision in this case.
Not sure how this works. Do I need to resubmit the pull request?

Comment (asimrp) [inline gpMgmt/bin/lib/gpcheckcat]: > On Dec 21, 2015, at 2:13 AM, Stephen Carter <notifications@github.com> wrote:
> 
> Not sure how this works. Do I need to resubmit the pull request?
> 

No, you should update the pull request with additional commits. But before pushing to master, make sure you squash all the additional commits.

**Comment by danielgustafsson** [comment]
Looks ready for merging, can you squash the commits into a single one?

**Comment by stephendotcarter** [comment]
I accidentally committed twice but the latest commit should have all the changes.

**Comment by hlinnaka** [comment]
@danielgustafsson, did you plan to pick this up, or should we find someone else to commit this?

**Comment by danielgustafsson** [comment]
My bad, I thought the submitter had access and would push it. I will go ahead and commit this.

**Comment by danielgustafsson** [comment]
Pushed to master in 4339a3b with a slightly reformatted commit message. Thanks!

**Pull request closed without merging**